src/Components/ExtractLevelsMenu.jsx

```diff
@@ -0,0 +1,165 @@
+
+import React, {useState, useEffect} from 'react'
+import Menu from '@mui/material/Menu'
+import MenuItem from '@mui/material/MenuItem'
+import {makeStyles} from '@mui/styles'
+import {TooltipIconButton} from './Buttons'
+import useStore from '../store/useStore'
+import {getModelCenter} from '../utils/cutPlane'
+import {addHashParams, getHashParams} from '../utils/location'
+import {Vector3} from 'three'
+import {useLocation} from 'react-router-dom'
+import {removePlanes} from '../utils/cutPlane'
+import LevelsIcon from '../assets/2D_Icons/Levels.svg'
+import {extractHeight} from '../utils/extractHeight'
+
+/**
+ * BasicMenu used when there are several option behind UI button
+ * show/hide from the right of the screen.
+ *
+ * @param {Array} listOfOptions Title for the drawer
+ * @return {object} ItemPropertiesDrawer react component
+ */
+export default function ExtractLevelsMenu({listOfOptions, icon, title}) {
+  const [anchorEl, setAnchorEl] = useState(null)
+  const classes = useStyles()
+  const open = Boolean(anchorEl)
+
+  const model = useStore((state) => state.modelStore)
+  let [floorplanMenuItems, showExtractMenu] = useState([])
+
+
+  const PLANE_PREFIX = 'p'
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget)
+  }
+  const handleClose = () => {
+    setAnchorEl(null)
+  }
+  const viewer = useStore((state) => state.viewerStore)
+  const location = useLocation()
+  const createFloorplanPlane = (h1, h2) => {
+    console.log('Got here')
+    removePlanes(viewer)
+    const modelCenter1 = new Vector3(0, h1, 0)
+    const modelCenter2 = new Vector3(0, h2, 0)
+    const normal1 = new Vector3(0, 1, 0)
+    const normal2 = new Vector3(0, -1, 0)
+    viewer.clipper.createFromNormalAndCoplanarPoint(normal1, modelCenter1)
+    viewer.clipper.createFromNormalAndCoplanarPoint(normal2, modelCenter2)
+  }
+  const planView = () => {
+    // viewer.context.ifcCamera.projectionManager.setOrthoCamera()
+    viewer.context.ifcCamera.toggleProjection()
+    viewer.plans.moveCameraTo2DPlanPosition(true)
+  }
+  const createPlane = (normalDirection) => {
+    const modelCenter = getModelCenter(model)
+    const planeHash = getHashParams(location, 'p')
+    console.log('in the function modelCenter', modelCenter)
```

Comment:
remove.

Reply:
Remove the entire ``createPlane`` function?

Reply:
There should be no console.logs in production

Reply:
Already addressed in #356. This commit was branched from #356, and after implementation of the auto-level extraction, #364 and #356 were rebased. Code cleanup took place later in #356.